node/README.md

```diff
@@ -0,0 +1,7 @@
+Add libraries meeded for gnupg. First execute the build command, then you have the rest of the commands availabe to you.
```

Comment:
Please follow the template we have for [nginx](https://github.com/unikraft/static-pie-apps/blob/master/nginx/README.md) for the README file.